package.json

```diff
@@ -146,5 +148,19 @@
   },
   "engines": {
     "node": ">= 8.3"
+  },
+  "dependenciesMeta": {
```

Comment:
these 4 are for `weak-napi`

Reply:
Solved upstream

Reply:
If I remove this, I get the same error I had before unplugging stuff. I updated the version I had by `yarn set version berry`, which said it downloaded from `master`

Reply:
Try with `yarn set version from sources`

Reply:
that worked, thanks!